src/views/splash/l10n.json

```diff
@@ -38,14 +38,16 @@
     "teacherbanner.classesButton": "My Classes",
     "teacherbanner.faqButton": "Teacher Account FAQ",
 
-    "hocbanner.title": "Get Creative with Coding!",
```

Comment:
What we try to do in these situations is to leave the existing phrase and key in place, and create a new one. That makes more sense with Transifex, so that if we ever want to use the old phrase, its translations are still in place and ready to go.